pkg/network/render.go

```diff
@@ -338,11 +335,11 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec) error {
 		if i < len(singleStack.ClusterNetwork) {
 			if !reflect.DeepEqual(singleStack.ClusterNetwork[i], dualStack.ClusterNetwork[i]) {
 				// Changed or re-ordered an existing ClusterNetwork element
-				return errors.Errorf("cannot change ClusterNetwork")
+				return errors.Errorf("cannot change primary ClusterNetwork when migrating to/from dual-stack")
 			}
 		} else if utilnet.IsIPv6CIDRString(dualStack.ClusterNetwork[i].CIDR) == EntryZeroIsIPv6 {
 			// Added a new element of the existing IP family
-			return errors.Errorf("cannot change ClusterNetwork")
+			return errors.Errorf("cannot append old IP family to ClusterNetwork list during dual-stack migration")
```

Comment:
how about "cannot add additional ClusterNetwork values of original IP family when migrating to dual stack" ?